edxval/tests/test_api.py

```diff
@@ -1373,19 +1376,16 @@ def test_video_image_urls_field_validation(self):
         # expect a validation error if we try to a list with non-string items
         with self.assertRaises(ValidationError) as set_exception:
             video_image.generated_images = ['a', 1, 2]
+            video_image.save()
 
         self.assertEqual(set_exception.exception.message, u'list must only contain strings.')
 
         # expect a validation error if we try to set non list data
         exception_messages = set()
-        for item in ('a string', 555, {'a': 1}, (1,), video_image):
+        for item in ('a string', 555, {'a': 1}, (1,)):
             with self.assertRaises(ValidationError) as set_exception:
                 video_image.generated_images = item
-
-            exception_messages.add(set_exception.exception.message)
-
-        self.assertEqual(len(exception_messages), 1)
-        self.assertEqual(exception_messages.pop(), u'Must be a valid list of strings.')
```

Comment:
@iloveagent57 If this exception is not raised now then we need to remove https://github.com/edx/edx-val/pull/81/files#diff-bf00c7cdc19274a54f59d0526f065c8dL236